docs/v2/2_installation.md

```diff
@@ -9,31 +9,27 @@ sidebar_label: Installation
 
 :::info
 
-In this section, we will go through the steps necessary to install the web3.unity SDK into the Unity3D game engine.
+In this section, we will go through the steps necessary for installing the web3.unity SDK into the Unity3D game engine. We will follow the installation with an example of how to implement a sample web3 login screen.
 
 :::
 
+## Installing web3.unity
+
 ### Install MetaMask
 
-You will need to have MetaMask installed in order to make web3-enabled games. If you already have MetaMask installed, you can skip this section and navigate to the [Downloading SDK](#downloading-the-sdk) section. If you have not installed a web3 wallet, please visit [MetaMask](https://t.co/8UNUrkShG6) for installation instructions.&#x20;
+You will need to have MetaMask & the appropriate browser extension installed in order to make web3-enabled games. If you already have MetaMask installed, you can skip this section and navigate to the [Downloading The SDK](#downloading-the-sdk) section. If you have not installed a web3 wallet, please visit [MetaMask](https://t.co/8UNUrkShG6) for installation instructions.&#x20;
 
 ### Downloading The SDK
 
-You will first need to download the web3.unity SDK from our GitHub repository. This can be found by looking for the web3.unity repo by following [this link](https://github.com/chainsafe).&#x20;
-
-![](v2Assets/step1.png)
-
-After clicking on the web3.unity link, find the **Releases** section. 
+In order to install web3.unity into the Unity3D game engine, you must download the `web3.unitypackage` file to your computer.
 
-![](v2Assets/step2.png)
-
-Here, you will be able to download the **web3.unitypackage** to your computer, which you will use to install into the Unity3D game engine.
+You can download the SDK with this `web3.unitypackage` [direct link](https://github.com/ChainSafe/web3.unity/releases/latest/download/web3.unitypackage), or visit our GitHub to view web3.unity's [latest release](https://github.com/ChainSafe/web3.unity/releases).
 
 ![](v2Assets/step3.png)
 
 ### Creating A New Unity3D Project
 
-Once you have downloaded the web3.unitypackage, you will need to create a new Unity3D project. In this example, we will create an empty 3D project. The SDK / Unity plugin supports all Unity project types. In this example, we will use a 3D scene.
+Once you have downloaded the `web3.unitypackage` file to your computer, you will need to create a new Unity3D project. In the following example, we will create an empty 3D project with a 3D scene. However, the web3.unity plugin supports all Unity project types. 
```

Comment:
Replace with:

Once you have downloaded the `web3.unitypackage` bundle to your computer, you will need to create a new Unity3D project. In the following example, we will create an empty 3D project with a 3D scene. However, the web3.unity plugin supports all Unity project types.

Reply:
fixed